src/ast/mod.rs

```diff
@@ -650,6 +650,8 @@ pub enum Expr {
         left: Box<Expr>,
         compare_op: BinaryOperator,
         right: Box<Expr>,
+        // ANY and SOME are synonymous
```

Comment:
Is there any one of the supported dialects supporting the SOME variant?

Reply:
I believe hive does: https://docs.cloudera.com/cdw-runtime/cloud/using-hiveql/topics/hive_comparison_predicates.html?

Reply:
Ah cool! Can we include the link in the comment here as well? It could help later on in case someone comes across this and wonders the same